### PR TITLE
Restore FroalaEditor extendability

### PR DESCRIPTION
**Pull request by mjauvin**
Fix for #4354

When forcing the "pluginsEnabled" Froala Editor option, this breaks Froala Editor extendability

### CODE REVIEW

File: modules/backend/formwidgets/richeditor/assets/js/richeditor.js [hunk 1]
```diff
@@ -168,11 +168,9 @@
             ? Infinity
             : $('.height-indicator', this.$el).height()
 
-        var enabledPlugins = ['align', 'audio', 'codeBeautifier', 'codeView', 'colors', 'draggable', 'emoticons', 'entities', 'file', 'fontFamily', 'fontSize', 'fullscreen', 'image', 'lineBreaker', 'link', 'lists', 'paragraphFormat', 'paragraphStyle', 'quote', 'table', 'url', 'video', 'pageLinks', 'figures']
-        if (this.options.useMediaManager) {
-            enabledPlugins.push('mediaManager')
+        if (! this.options.useMediaManager) {
```
Comment (LukeTowers) [inline modules/backend/formwidgets/richeditor/assets/js/richeditor.js]: no space between `!` and `this`

**Comment by mjauvin** [comment]
@NathanBastiaans can you confirm if this PR solves your issue? I tested it locally with your procedure to reproduce and it works.

**Comment by NathanBastiaans** [comment]
Did a couple of tests with my froala plug-in and everything seems to work fine!

**Comment by mjauvin** [comment]
@LukeTowers can you merge this asap, this is currently breaking all plugins extending the RichEditor.

Thanks.

**Comment by w20k** [comment]
Tested as well, seems like fixed the issue.

**Pull request merged**